specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/secureScore.json

```diff
@@ -0,0 +1,615 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoreControlDefinitionItem"
+          }
+        },
+        "nextLink": {
+          "description": "The URI to fetch the next page.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SecureScoreControlDefinitionItem": {
+      "description": "Secure Score Control's Definition information",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecureScoreControlDefinitionItemProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecureScoreControlDefinitionItemProperties": {
+      "description": "Secure Score Control Definition Properties",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string"
+        },
+        "description": {
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string",
+          "maxLength": 256
+        },
+        "maxScore": {
+          "description": "Maximum control score (0..10)",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 10,
+          "exclusiveMinimum": false
+        },
+        "source": {
+          "description": "Source object from which the control was created",
+          "readOnly": true,
+          "$ref": "#/definitions/SecureScoreControlDefinitionSource"
+        },
+        "assessmentsDefinition": {
+          "description": "array of assessments metadata ids",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoreControlDefinitionAssessmentsId"
+          }
+        }
+      }
+    },
+    "SecureScoreControlDefinitionAssessmentsId": {
+      "type": "object",
+      "description": "Secure Score Control Definition Properties",
+      "properties": {
+        "id": {
+          "description": "Full assessment metadata azure ids that build the control",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SecureScoreControlDefinitionSource": {
+      "description": "representing the source of the control",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "sourceType": {
+          "description": "BuiltIn if the control is built-in from Azure Security Center managed assessments, Custom if the assessment based on custom Azure Policy definition, CustomerManaged for customers who build their own controls",
+          "type": "string",
+          "enum": [
+            "BuiltIn",
+            "CustomPolicy",
```

Comment:
remove if it will not be published in first step

Reply:
Removing it. To be added once Custom controls are added